api/schema_test.go

```diff
@@ -0,0 +1,65 @@
+package api
+
+import (
+	"context"
+	"errors"
+	"github.com/gojuno/minimock/v3"
+	"github.com/insolar/insolar/api/seedmanager"
+	"github.com/insolar/insolar/configuration"
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/insolar/pulse"
+	"github.com/insolar/insolar/testutils"
+	"github.com/insolar/rpc/v2"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"testing"
+)
+
+func TestSchemaService_Get(t *testing.T) {
+	defer testutils.LeakTester(t)
+
+	availableFlag := false
+	mc := minimock.NewController(t)
+	checker := testutils.NewAvailabilityCheckerMock(mc)
+	checker = checker.IsAvailableMock.Set(func(ctx context.Context) (b1 bool) {
+		return availableFlag
+	})
+
+	// 0 = false, 1 = pulse.ErrNotFound, 2 = another error
+	pulseError := 0
+	accessor := mockPulseAccessor(t)
+	accessor = accessor.LatestMock.Set(func(ctx context.Context) (p1 insolar.Pulse, err error) {
+		switch pulseError {
+		case 1:
+			return insolar.Pulse{}, pulse.ErrNotFound
+		case 2:
+			return insolar.Pulse{}, errors.New("fake error")
+		default:
+			return *insolar.GenesisPulse, nil
+		}
+	})
+	cfg := configuration.NewAPIRunner(false)
+	cfg.Address = "localhost:19192"
+	cfg.SwaggerPath = "testdata/api-exported.yaml"
```

Comment:
pls add case with no .yaml provided

Reply:
it will die before in validator initialization